lib/assembly-objectfile/object_fileable.rb

```diff
@@ -127,12 +128,21 @@ def mimetype
         mimetype = ''
         mime_type_order.each do |mime_type_method|
           mimetype = public_send("#{mime_type_method}_mimetype") if VALID_MIMETYPE_METHODS.include?(mime_type_method)
-          break if !mimetype.nil? && mimetype != ''
+          break if mimetype.present?
```

Comment:
rubocop, i presume 🤣

Reply:
surprisingly rubocop didn't complain, but i saw it and decided to change it